valhalla/mjolnir/osmway.h

```diff
@@ -2663,8 +2679,9 @@ struct OSMWay {
   uint16_t use_sidepath_ : 1;
   uint16_t bike_forward_ : 1;
   uint16_t bike_backward_ : 1;
-  bool lit_ : 1;
```

Comment:
that was a fail in the `lit` PR earlier this year. luckily only on this temp structure